frontend/src/formdescriptioneditor/BarChartWidget.tsx

```diff
@@ -0,0 +1,82 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import { BarChartComponent } from 'charts/barChart/BarChartComponent';
+import { BarChart } from 'charts/Charts.types';
+import { WidgetProps } from 'formdescriptioneditor/WidgetEntry.types';
+import React, { useEffect, useRef, useState } from 'react';
+
+const useStyles = makeStyles((theme) => ({
+  selected: {
+    color: theme.palette.primary.main,
+  },
+}));
+
+export const BarChartWidget = ({ widget, selection }: WidgetProps) => {
+  const classes = useStyles();
+  const chart: BarChart = {
+    id: 'notUsed',
+    metadata: null,
```

Comment:
The interface `Chart` seems to require metadata. Add arbitrary data here.

Reply:
ok